buzzbooster-ios/BSTSample/AppDelegate.swift

```diff
@@ -0,0 +1,60 @@
+import UIKit
+import BuzzBooster
+import UserNotifications
+import Toast
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate {
+  static var APP_KEY: String = "0d944ac8a05b"
+  static var USER_ID: String = NSUUID().uuidString
+
+  var window: UIWindow?
+  
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    UNUserNotificationCenter.current().delegate = self;
+    let notificationOptions = BSTNotificationOptions { builder in
+      builder.userInfo = ["sample_key":"sample_value"]
+      builder.categoryIdentifier = "sample_category"
+    }
+    let config = BSTConfig { builder in
+      builder.appKey = AppDelegate.APP_KEY
+      builder.notificationOptions = notificationOptions
+    }
+    BuzzBooster.initialize(with: config)
+    BuzzBooster.startService()
+    
+    window = UIWindow(frame: UIScreen.main.bounds)
+    let navigationViewController = UINavigationController.init(rootViewController: ViewController())
```

Comment:
nit. Swift에서는 `.init`을 생략하는 것이 일반적입니다 🙂 [style guide](https://google.github.io/swift/#initializers)
```suggestion
    let navigationViewController = UINavigationController(rootViewController: ViewController())
```